libvast/src/format/arrow.cpp

```diff
@@ -15,15 +15,14 @@
 
 #if VAST_ENABLE_ARROW
 
-#  include "vast/format/arrow.hpp"
-
```

Comment:
Didn't we once have an `IncludeCategory` for this?

Reply:
We still do. I think this is a change in recent clang-format versions. Let's just leave it as formatted by clang-format here.